ui/app/src/modules/Preview/PreviewConcierge.tsx

```diff
@@ -461,27 +463,22 @@ export function PreviewConcierge(props: any) {
           nou(audiencesPanel.model) &&
           nou(audiencesPanel.error)
         ) {
+          //TODO: Jose Vega re-render when this panel is opened and site is changed;
```

Comment:
What's the meaning of this?

Reply:
this means that the audiencesPanel needs changes to support the re-render when the site changes on the address bar